opsdroid/core.py

```diff
@@ -101,19 +111,25 @@ def default_connector(self):
         return default_connector
 
     def exit(self):
-        """Exit application."""
+        """Exit application.
+        
+        """
         _LOGGER.info(_("Exiting application with return code %s"), str(self.sys_status))
         sys.exit(self.sys_status)
 
     def critical(self, error, code):
-        """Exit due to unrecoverable error."""
+        """Exit due to unrecoverable error.
+        
```

Comment:
Same thing here, critical has `error` and `code` arguments, you should add these to the docstring and explain what this method does - basically it will exit opsdroid due to an unrecoverable error, logs the error message to logging and then exits.